java-agent/instrumentation/dropwizard/src/test/groovy/DropwizardTest.groovy

```diff
@@ -29,9 +30,11 @@ class DropwizardTest extends HttpServerTest<DropwizardTestSupport, Servlet3Decor
 
   @Override
   DropwizardTestSupport startServer(int port) {
+    println "Port: $port"
     def testSupport = new DropwizardTestSupport(testApp(),
       null,
-      ConfigOverride.config("server.applicationConnectors[0].port", "$port"))
+      ConfigOverride.config("server.applicationConnectors[0].port", "$port"),
+      ConfigOverride.config("server.adminConnectors[0].port", "" + PortUtils.randomOpenPort()))
```

Comment:
The `check` build failed, flagging this line, see https://829-210933087-gh.circle-artifacts.com/0/home/circleci/dd-trace-java/reports/java-agent/instrumentation/dropwizard/codenarc/test.html

Reply:
OK. That's a lazy way of making a string from an int. Not that it matters in this context, but we want clean builds, so I'll change it!